packages/cloud_firestore/cloud_firestore/ios/cloud_firestore.podspec

```diff
@@ -24,9 +24,5 @@ Firestore plugin for Flutter.
   s.dependency 'Firebase/Firestore', '~> 6.0'
   s.static_framework = true
 
-  s.prepare_command = <<-CMD
-      echo // Generated file, do not edit > Classes/UserAgent.h
-      echo "#define LIBRARY_VERSION @\\"#{libraryVersion}\\"" >> Classes/UserAgent.h
-      echo "#define LIBRARY_NAME @\\"flutter-fire-fst\\"" >> Classes/UserAgent.h
-    CMD
+  s.pod_target_xcconfig = { 'GCC_PREPROCESSOR_DEFINITIONS' => '$(inherited) LIBRARY_VERSION=@\"' + libraryVersion + '\" LIBRARY_NAME=@\"flutter-fire-fst\"' }
```

Comment:
Does this actually work?  I wouldn't have expected any @ signs everywhere.

Reply:
I was expecting something like (use Ruby string interpolation):
```
s.pod_target_xcconfig = { 'GCC_PREPROCESSOR_DEFINITIONS' => '$(inherited) LIBRARY_VERSION="#{libraryVersion}" LIBRARY_NAME=flutter-fire-fst' }
```

Reply:
Oh I see
> It wasn't converting the `#{libraryVersion} ` to the actual value so I concatenated the string.

If I recall it only works in double quotes:
```ruby
"'$(inherited) LIBRARY_VERSION=\"#{libraryVersion}\" LIBRARY_NAME=flutter-fire-fst"
```

Reply:
Need the @ to psudeo string-ize the the macro

Reply:
We both tried a bunch of options offline, but it's still giving some errors. We'll continue on Monday.